### PR TITLE
Overhaul videos.

### PR DESCRIPTION
**Pull request by kylorhall**
Part of a few Vimaly Stories, eg. [Core Web Vitals](https://vimaly.com/#j8mqm/t/www_seo_5._Core_Web_Vitals_-_pages_with_poor_LCPCL.../msSqFn0aDat9Pi4eR).

See: https://github.com/vimeo/player.js

Previously, the player would be loaded via Froogaloop after the page was loaded.  It was a bit hacked together.

 - Replace Froogaloop with Vimeo directly.  Eg. https://github.com/vimeo/player.js/blob/master/docs/migrate-from-froogaloop.md
 - Use `require.js` to asynchronously load this.
 - Preload this as well.
 - Change from a raw iframe to just requiring an id.
 - Fixes colors being wrong and title/portraits being inconsistent.
 - These may have actually been broken due to now `allow="autoplay"` :scream:
 - Minor styling change.

### CODE REVIEW

File: source/css/elements/_video.scss [hunk 1]
```diff
@@ -10,25 +10,28 @@
...
+    left: 0;
+    width: 100%;
+    height: 100%;
+    border: none;
+  }
 }
 
+
 .video_overlay {
+  background: rgba(#000, 0.12); // if we don't have a background image
```
Comment (kyle-beattie) [inline source/css/elements/_video.scss:26]: Is this SCSS syntax?

```suggestion
  background: rgba(0, 0, 0, 0.12); // if we don't have a background image
```

File: source/page-xero.html.erb [hunk 2]
```diff
@@ -70,14 +70,13 @@ section_class: 'section_home'
     </div>
   </section>
 
-  <% if locale_obj[:xero_cashbook] %>
+  <% if true || locale_obj[:xero_cashbook] %>
```
Comment (kyle-beattie) [inline source/page-xero.html.erb:73]: Wouldn't this always evaluate to `true`? I.e `true || false === true`?

File: source/css/elements/_video.scss (hunk 1 above)
Comment (kylorhall) [reply source/css/elements/_video.scss]: Yeah, SCSS.  Works, but I agree.

File: source/page-xero.html.erb (hunk 2 above)
Comment (kylorhall) [reply source/page-xero.html.erb:73]: No, this was me debugging...thank you!

```suggestion
  <% if locale_obj[:xero_cashbook] %>
```

File: source/js/src/video.js.es6 [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    });
+
+    // If we have an overlay (eg. a background + play button, we interact with that instead)
+    const playOverlay = vimeoSrc.parentNode.querySelector('.video_overlay');
+    if (playOverlay) {
+      vimeo.on('play', function() {
+        playOverlay.style.display = "none";
+      });
+
+      playOverlay.addEventListener("click", function(e) {
```
Comment (kyle-beattie) [inline source/js/src/video.js.es6:29]: Style thing - got some double quotes here:

```suggestion
        playOverlay.style.display = 'none';
      });

      playOverlay.addEventListener('click', function(e) {
```

**Review by kyle-beattie — APPROVED** [review]
Looks good 👍  Haven't seen require.js in a long time!

**Pull request merged**